css/soe_helper.css

```diff
@@ -0,0 +1,614 @@
+
+@media (max-width: 480px) {
+  #header #logo.logo-mobile {
+    display: table-cell;
+    padding: 0 10px 0 0;
+    border-right: 1px solid !important;
+  }
+}
+
+@media (max-width: 480px) {
+  #name-and-slogan {
+    display: table-cell;
+  }
+  #name-and-slogan.with-logo {
+    padding: 0 0 0 10px;
+  }
+  #site-title-first-line.site-title-uppercase {
+    font-size: 28px;
+  }
+}
+
+
+/* SIDEBAR FIRST AND MAIN CONTENT GUTTER */
+
+@media (min-width: 1200px) {
+  .span9 {
+    width: 740px;
+  }
+  #content.span9 {
+    margin-left: 80px;
+  }
+}
+
+
+/* MAIN NAVIGATION STYLES */
+
+#main-menu .nav-collapse.collapse {
+  float: right;
+  margin-top: -46px;
+}
+
+.navbar .nav > li > a {
+  font-size: 1.1em;
+  font-weight: 400;
+  border-bottom: 2px solid transparent;
+  padding-bottom: 30px;
+}
+
+.navbar .nav > li > a:focus,
+.navbar .nav > li > a:hover,
+.navbar .nav > .active-trail > a {
+  border-bottom: 2px solid #333333;
+  padding-bottom: 30px;
+}
+
+.navbar {
+  margin-bottom: 0;
+}
+
+.region-navigation .block.contextual-links-region {
+  float: left;
+  padding-right: 7px;
+  margin-right: 7px;
+}
+
+.site-main-menu {
+  background: #fff;
+  border-bottom: 1px solid #e1e1e1;
+}
+
+.site-main-menu ul li.last.leaf {
+  height: 31px;
+  border-right: 1px solid #d6d6d6;
+  padding-right: 1em;
+}
+
+@media (max-width: 1200px) {
+  #main-menu .nav-collapse.collapse {
+    float: left;
+    margin-top: 0;
+  }
+}
+
+@media (max-width: 767px) {
+  .container {
+    margin-bottom: 10px;
+  }
+  .site-main-menu ul li.last.leaf {
+    height: auto;
+    border-right: none;
+    padding-right: 0;
+  }
+  .btn.btn-navbar {
+    float: right;
+  }
+  .navbar .btn-navbar {
+    margin-top: -50px;
+  }
+  .nav-collapse,
+  .nav-collapse.collapse {
+    width: 100%;
+  }
+}
+
+@media (max-width: 480px) {
+  .navbar .btn-navbar {
+    margin-top: -57px;
+  }
+}
+
+@media (max-width: 385px) {
+  .btn.btn-navbar {
+    float: left;
+  }
+  .navbar .btn-navbar {
+    margin-top: 0;
+  }
+}
+
+
+/* SUB-NAVIGATION STYLES */
+
+.sidebar .block-menu-block .menu-block-wrapper > .menu > li.last {
+  border-bottom: none;
+}
+
+.sidebar .block-menu-block .menu-block-wrapper > .menu > li > a {
+  font-weight: 400;
+}
+
+
+/* EXPANDABLE SEARCH */
+
+#block-stanford-search-api-search-api-search-block h2,
+.navbar #block-stanford-search-api-search-api-search-block input.btn-search {
+  display: none;
+}
+
+input[type="text"] {
+  box-shadow: none;
+}
+
+input::-webkit-input-placeholder {
+  color: transparent;
+}
+
+input::-moz-placeholder {
+  color: transparent;
+}
+
+#block-stanford-search-api-search-api-search-block {
+  float: right;
+}
+
+#block-stanford-search-api-search-api-search-block input.search-query {
+  width: 15px;
+  padding: 7px 9px 7px 9px;
+  margin-top: -6px;
+  color: transparent;
+  cursor: pointer;
+  background: #ffffff url(https://sites.stanford.edu/jse-soe-new/sites/default/files/soe_magnifying-glass.png) no-repeat 6px center;
```

Comment:
Thanks for catching this! This was a result of having copied the injector file (where testing was being done with an image uploaded locally). I will add an /img file to the helper module and call the image accordingly.